src/main/java/net/teamuni/cashmf/CashMF.java

```diff
@@ -14,18 +17,35 @@
 
 public class CashMF extends JavaPlugin {
     private static CashMF instance;
-    private static PlayerConf playerConf;
+
+    private static Conf conf;
     private static MessageConf messageConf;
 
+    private static PlayerConf playerConf;
+    private static SQL sql;
+    private static MongoDB mongoDB;
+
     private SkriptAddon addon;
 
     @Override
     public void onEnable() {
         instance = this;
 
         // *.yml 파일 설정
-        playerConf = new PlayerConf();
         messageConf = new MessageConf();
+        conf = new Conf();
+
+        // 데이터 저장 방식 설정
+        if (conf.database.get("type").equals("yaml")) {
+            playerConf = new PlayerConf();
+
+        } else if (conf.database.get("type").equals("mongodb")) {
+            mongoDB = new MongoDB();
+
+        } else {
+            sql = new SQL();
+        }
```

Comment:
3개의 클래스 모두 공통적으로 load 와 save 메소드가 사용됩니다. load 와 save 메소드를 가지고 있는 인터페이스를 하나 만들어서 캡슐화 시켜버리면 좋겠네요.
Database라는 인터페이스를 만들어 캡슐화한 예시
```suggestion
        Database database;
        if (conf.database.get("type").equals("yaml")) {
            database = new PlayerConf();
        } else if (conf.database.get("type").equals("mongodb")) {
            database = new MongoDB();
        } else {
            database = new SQL();
        }
        ...
        getDatabase().save()
        getDatabase().save()
```